lib/LANraragi/Model/Reader.pm

```diff
@@ -113,9 +113,6 @@ sub build_reader_JSON ( $self, $id, $force ) {
 
     return {
         pages => \@images_browser,
-
-        # filesizes => \@sizes,
-        job => $jobid
```

Comment:
Ichaival (when force refreshing) and LRReader (always) use this to properly wait for extraction, Ichaival silently fails and LRReader shows an error message (as it is expected to be present).